DNN Platform/Library/Services/ImprovementsProgram/BeaconService.cs

```diff
@@ -1,196 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information
-
-namespace DotNetNuke.Services.ImprovementsProgram
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Configuration;
-    using System.Linq;
-    using System.Security.Cryptography;
-    using System.Text;
-    using System.Web;
-
-    using DotNetNuke.Application;
-    using DotNetNuke.Common.Utilities;
-    using DotNetNuke.Entities.Host;
-    using DotNetNuke.Entities.Portals;
-    using DotNetNuke.Entities.Tabs;
-    using DotNetNuke.Entities.Users;
-    using DotNetNuke.Framework;
-
-    public class BeaconService : ServiceLocator<IBeaconService, BeaconService>, IBeaconService
```

Comment:
For safety's sake, I think I'd prefer that this be marked obsolete and gutted, rather than removed entirely.

Reply:
How strongly do you feel about this?  Its all internal usage, it sis something that SHOULDN"T be used by third-party.  But I can add it back in with an obsolete flag if needed

Reply:
I'm happy to be overruled if no one else thinks it's important.  I could see a third party using it as a proxy for whether they are going to send their own beacon.  I don't know of any actual usages, though, it's just a theory.

Reply:
Yeah, I am a bit on the fence, but I think this should just have never been made public in the first place and I highly doubt it is used much by third parties. So I am leaning on the side of removing it...